eng/pipelines/templates/steps/sync-repo-branch.yml

```diff
@@ -1,110 +0,0 @@
-parameters:
```

Comment:
There are other usages of this template like https://dev.azure.com/azure-sdk/internal/_apps/hub/ms.vss-build-web.ci-designer-hub?pipelineId=1767&nonce=fwPIHOo9x%2B4z6qQGGQqt4Q%3D%3D&branch=master we cannot remove this without also updating other places. I suggest keeping the sync-repo-branch.yml file and the parameters consistent.

Reply:
thanks for reminding. Revert it ~